### DISCUSSION TITLE
Feature Request - Change the tray icon color according to active power mode

### DISCUSSION THREAD

**Discussion by fscanp** [seerge/g-helper#1131] [Ideas]
Would be nice to be able to tell which power mode is currently selected just by glancing at the tray icon color. The default green/blue/red mode color scheme is just fine as long as it's reflected in the tray icon as well.

**Comment by seerge** [comment]
Icon color is already used to indicate GPU mode

**Reply by fscanp** [reply]
Could you extend function that to include the power modes as well?

Honestly I don't change GPU mode that often which is why i didn't realize the icon color is tied to it. The power modes, however, I switch around more often between silent and the turbo when I run processor heavy tasks. 

Not super important, but would help a bit as I sometimes forgot to switch modes when running demanding apps until I see it running sluggishly

Thanks.
